example_1.py

```diff
@@ -1,117 +0,0 @@
-# This example presents a very simple Cee section,
-# solved for pure compression,
-# in the Imperial unit system
```

Comment:
Why did you delete the examples? I think they're a very important part of users understanding how to use pyCUFSM. They certainly don't do any harm. I also plan to use them in automated validation testing of pyCUFSM. Unless you can give a really good reason for deleting them, can you please restore them?

Reply:
Didn't mean to delete..didn't want to commit the changes I made into examples 1 and 2 into the main repo so didn't commit into mine and ended up deleting from mine..
Will add it back

Reply:
It may be easier to create a new branch on _your_ fork, and pull request that _branch_ into here. Right now, you're pull requesting your fork's `master` into here, which may make it a bit more difficult to exclude changes you don't want to commit. (This doesn't matter to me - just may be easier for you to work with. I have a couple upstream projects I contribute to for which I do this)